eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ExternalDependencyExposedCheck.java

```diff
@@ -21,16 +21,13 @@
  *  No external dependency exposed in public API
  */
 public class ExternalDependencyExposedCheck extends AbstractCheck {
-    private static final String EXTERNAL_DEPENDENCY_ERROR =
-        "Class ''%s'', is a class from external dependency. You should not use it as a return or method argument type.";
-
     private static final Set<String> VALID_DEPENDENCY_SET = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
         "java", "com.azure", "reactor", "io.netty.buffer.ByteBuf"
     )));
 
     private final Map<String, String> simpleClassNameToQualifiedNameMap = new HashMap<>();
 
-    private boolean isPublicClass;
+    private boolean isPublicAPI;
```

Comment:
tbf, the name before worked. public/protected are publicly accessible classes. a class isn't an API